libs/application/templates/transport-authority/change-co-owner-of-vehicle/src/fields/Overview/index.tsx

```
@@ -107,11 +113,11 @@ export const Overview: FC<
           })
 
           if (resSubmit?.data) {
-            setLoading(false)
+            setButtonLoading(false)
             setStep && setStep('conclusion')
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)